crates/oxc_linter/src/snapshots/no_extra_boolean_cast.snap

```diff
@@ -0,0 +1,2498 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 80
+expression: no_extra_boolean_cast
+---
+  ⚠ eslint(no-extra-boolean-cast): Redundant double negation
+   ╭─[no_extra_boolean_cast.tsx:1:1]
+ 1 │ if (!!foo) {}
+   ·      ────
```

Comment:
I noticed this is not covering the two `!`s, is it possible to span the two `!`s as pointed out by the message "double negation"?

Reply:
i.e.

```
 if (!!foo) {}
     ─────

```

Reply:
Thanks for catching! fixed